string joint_values
geometry_msgs/PoseStamped GraspPose
float64 MinDist
float64 Volume
